_episodes/10-data-transformation.md

```diff
@@ -38,9 +38,43 @@ So far we've been looking only at 'String' type data. Much of the time it is pos
 >3. Note how the values are now displayed in green and follow a standard convention for their display format (ISO 8601) - this indicates they are now stored as date data types in OpenRefine. We can now carry out functions that are specific to Dates
 >4. On the Date column dropdown select ```Edit column->Add column based on this column```. Using this function you can create a new column, while preserving the old column
 >5. In the 'New column name' type "Formatted-Date"
->6. In the 'Expression' box type the GREL expression ```value.toString("dd MMMM yyyy")```
+>6. In the 'Expression' box type the GREL expression ```value.toString("dd MMM yyyy")
```

Comment:
Is there a reason for changing "MMMM" to "MMM" in the date formatter here? These will give different outcomes (MMMM will use the full month name while MMM will use an abbreviated month name - e.g. 29 July 2022 vs 29 Jul 2022)

Reply:
Yes I understand! There was not real reason, So I changed it back to the original: ```value.toString("dd MMMM yyyy")```